commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/ComplexFunctions.java

```diff
@@ -354,6 +665,1057 @@ private static <R> R log(DoubleUnaryOperator log,
         return action.apply(re, arg(real, imaginary));
     }
 
+    /**
+     * Returns the square root of the complex number using its real
```

Comment:
This has not copied over the javadoc from Complex.

I just noticed the same is true for ComplexFunctions::abs too (from the last PR). Can you update that accordingly.